### DISCUSSION TITLE
"ESLint couldn't find the config "prettier" to extend from" When using lint-staged on windows.

### DISCUSSION THREAD

**Discussion by tautf** [eslint/eslint#15386] [Q&A] [answered]
When using eslint in combination with lint-staged and subfolders i receive the error:
`ESLint couldn't find the config "prettier" to extend from. Please check that the name of the config is correct.`

package.json for lint-staged:
```
"lint-staged": {
    "webclient/**/*.ts": "npm run webclient-lint",
    "webserver/**/*.ts": "npm run webserver-lint"
  },
  "scripts": {
    "prepare": "husky install",
    "webclient-lint": "cd webclient && npx eslint -c .eslintrc.json",
    "webserver-lint": "cd webserver && npx eslint -c .eslintrc.json"
  }
```

Running eslint directly in webserver or webclient folder works fine. `eslint-config-prettier` plugin is installed in both subfolders.

Any ideas how i could resolve this?

**Comment by nzakas** [comment, marked as answer]
You need eslint-config-prettier installed in the directory you’re running the script from. If it’s installed in a sub folder, ESLint can’t find it.

**Comment by Mouvedia** [comment]
Keep the main config in the parent and add .eslintrc
```json
{
  "extends": ["prettier"]
}
```
in both children folders.
